profile/README.md

```diff
@@ -0,0 +1,21 @@
+<h1 align="center">Run JavaScript <a href="https://nodejs.org/en/download">Everywhere</a>.®</h1><br>
+
+<p align="center">
+  <a href="https://nodejs.org/">
+    <img src="https://raw.githubusercontent.com/nodejs/nodejs.org/main/public/static/images/logo-hexagon.svg" alt="Bootstrap logo" height="140">
```

Comment:
```suggestion
    <img src="https://raw.githubusercontent.com/nodejs/nodejs.org/main/public/static/images/logo-hexagon.svg" alt="Node.js logo" height="140">
```

Reply:
Good catch! I 100% grabbed this from the Bootstrap org 🤡